module/Activity/Module.php

```diff
@@ -115,6 +114,7 @@ public function getServiceConfig()
 
                     $acl->allow('user', 'activity', 'create');
                     $acl->allow('user', 'activitySignup', ['signup', 'signoff', 'checkUserSignedUp']);
+                    $acl->allow('user', 'activity', 'viewDetails', new IsOrganMember());
```

Comment:
I'm wondering if it wouldn't make more sense to just have an organ member role which is automatically applied to all organ members. However that isn't really part of this PR.